components/topic/ImageGrid.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Container, Carousel } from "react-bootstrap";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+// TODO: Should accept an array of image links which are displayed here
+
+const ImageGroup = () => {
```

Comment:
Okay, yeah this is definitely still a todo, but my assumption was that if this is being used, it will always be four images (other wise they would just include images separately on the page). We could revisit this

Reply:
I also changed the file name to ImageGroup